modules/mongodb/charts.go

```diff
@@ -529,6 +571,31 @@ func (m *Mongo) removeDatabasesFromDBStatsCharts(newDatabases []string) {
 	}
 }
 
+// removeReplicaSetMember removes dimensions for not existing
+// replicate set members
+func (m *Mongo) removeReplicaSetMembers(newMembers []string) {
+	diff := sliceDiff(m.replSetMembers, newMembers)
+	for _, name := range diff {
+		dimID := replicationHeartbeatLatencyDimPrefix + name
+		err := m.charts.Get(replicationHeartbeatLatency).MarkDimRemove(dimID, true)
+		if err != nil {
+			m.Warningf("failed to remove dimension %s from replication_heartbeat_latency chart: %v", dimID, err)
```

Comment:
```suggestion
			m.Warningf("failed to remove dimension: %v", err)
```

Can be simplified, because the error contains the chart and dimension id's

https://github.com/netdata/go.d.plugin/blob/345a6a148141a72f48c747c40f4ab5c08a881791/agent/module/charts.go#L250-L253

Reply:
done